src/bittytax/conv/parsers/coinbase.py

```diff
@@ -220,7 +237,27 @@ def _do_parse_coinbase(
                 fee_asset=currency,
                 wallet=WALLET,
             )
-    elif row_dict["Transaction Type"] in ("Sell", "Advanced Trade Sell"):
+    elif row_dict["Transaction Type"] == "Sell":
+        # Saw a sell order for USD to USD? Looks like a bug, so check for these and skip.
```

Comment:
Noticed this too in my own transaction history report. Looks like Coinbase fixed one thing and broke another.

They are converting any fiat currency to the local currency before a performing a trade, it also means the actual trades are sometimes reported with the wrong currency used.

I can see lots of GBP to GBP trades, also EUR to GBP, but the trade was made with EUR not GBP.